src/vscode-bicep/src/visualizer/app/components/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import ReactDOM from "react-dom";
+import { createGlobalStyle } from "styled-components";
+
+import { App } from "./App";
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    font-family: Segoe UI, Helvetica Neue, Helvetica, sans-serif;
```

Comment:
Are there any guidelines from VSCode about fonts? Any way to match this up with the theme?

Reply:
AFAIK there's no guidelines for fonts to use in webviews. We can read `--vscode-editor-font-family` in a webview but that's the monospaced font for code. That being said, I found the following comment in VSCode's source code, and I think we can use the same fonts in our webview:

```typescript
/**
 * The best font-family to be used in CSS based on the platform:
 * - Windows: Segoe preferred, fallback to sans-serif
 * - macOS: standard system font, fallback to sans-serif
 * - Linux: standard system font preferred, fallback to Ubuntu fonts
 *
 * Note: this currently does not adjust for different locales.
 */
export const DEFAULT_FONT_FAMILY = isWindows ? '"Segoe WPC", "Segoe UI", sans-serif' : isMacintosh ? '-apple-system, BlinkMacSystemFont, sans-serif' : 'system-ui, "Ubuntu", "Droid Sans", sans-serif';
```

Reply:
If people ask, we can always expose as a setting.

Reply:
Yeah good pt.